brigade/core/configuration.py

```diff
@@ -0,0 +1,16 @@
+import os
+
+c = {
+    'ssh_config_file': os.environ.get('BRIGADE_SSH_CONFIG_FILE') or
+    os.path.join(os.path.expanduser("~"), ".ssh", "config"),
+}
+
+
+class Config:
+    """
+    Documentation
+
+    """
+
+    def __init__(self):
```

Comment:
I'd add also the option to pass a configuration file and load the config parameters from there.

Reply:
By the way, I guess order should be:

1. env variable
2. configuration file
3. default